firmware/powerboard2/code/app/I2C_componentSpecific.c

```diff
@@ -0,0 +1,21 @@
+/*
+ * I2C_componentSpecific.c
+ *
+ *  Created on: Dec 26, 2020
+ *      Author: Shuhang Tan
+ */
+
+#include "I2C_componentSpecific.h"
+#include "stm32f4xx.h"
+
+//To be updated
+static const uint8_t I2C_Address[I2C_CHANNEL_COUNT] = {

```

Comment:
I'd recommend setting up a config struct. Look at `debug.c` and `debug_componentSpecific.c` for an example.